# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../log4cxx/compile-fix.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur log4cxx-0.9.7-orig/include/log4cxx/xml/domconfigurator.h log4cxx-0.9.7/include/log4cxx/xml/domconfigurator.h
--- log4cxx-0.9.7-orig/include/log4cxx/xml/domconfigurator.h	2009-10-28 14:14:39.000000000 +0000
+++ log4cxx-0.9.7/include/log4cxx/xml/domconfigurator.h	2009-10-28 14:15:23.000000000 +0000
@@ -208,7 +208,7 @@
 				spi::LoggerRepositoryPtr& repository);
 
 		protected:
-			String DOMConfigurator::subst(const String& value);
+			String subst(const String& value);
 
 		protected:
 			void * appenderBag;
